.gitignore

```diff
@@ -13,4 +13,5 @@ venv/
 *.container
 *.image
 *.volume
-__pycache__/
\ No newline at end of file
+__pycache__/
+.aider*
```

Comment:
I actually used this aider tool to rewrite a lot of the boilerplate associated with introducing a new command.